config/README.rst

```diff
@@ -319,6 +324,10 @@ additional-envlist
   ``testenv-additional`` (see below). This option has to be a list of strings
   without indentation.
 
+testenv-extras
+  Replacement for the default ``extras`` option in ``[testenv]`` of
```

Comment:
Does it actually have to be a replacement? Having `docs` automatically be added if `--with-sphinx-doctests` is used is actually quite handy.

Reply:
Eh, I mainly went with a replacement to keep the option name shorter.  I can create a PR for `testenv-additional-extras` if you prefer.

(I'm strongly against packages assuming `testenv-additional` can be used to extend an arbitrary option that happened to be right before the place %(testenv-additional)s got expanded at some point in time.)

Reply:
I'd prefer `testenv-additional-extras`.

It was probably not such a good idea to use `testenv-additional` for additional extras in the first place.

Reply:
I've done that in #76.

Reply:
Thank you!